1_space_odyssey/js/camilla-9000/app.js

```diff
@@ -106,27 +106,90 @@ console.log(".-.-. Booting .-.-.");
 
 
+if (hasJetPack) {
+  console.log("JetPack is enabled!!")
+} else if (hasJetPack) {
+  console.log("JetPack NOT enabled, please enable jetPack before doing a space walk")
+}
 
 
 
-/* 
+/*
  2. BROKEN SATELLITE::::::
     You need to send the droid out on a space walk to fix the satellite, with out it you can't communicate with earth.
-    a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid 
-       cannot go outside, console log the correct log based on that condition. 
+    a) write an if condition checking if the jet pack is enabled, if true the droid can go outside if false the droid
+       cannot go outside, console log the correct log based on that condition.
 */
-if(hasJetPack) {
-    console.log("JetPack is enabled!!! off I go outside to fix the satellite")
-}
-else {
-    console.log("JetPack NOT enable, please enable jetPack before doing a space walk")
-}
 
-
-/* 
+/*
  uncomment these console logs and write your if else statement and use them to console log out the correct statement
 
  console.log("JetPack is enabled!!! off I go outside to fix the satellite");
  console.log("JetPack NOT enabled, please enable jetPack before doing a space walk");
 
  */
+/*
+ 3. METEOR SHOWER::::::
+    On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+    a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+      create an if condition in your loop to print only 1 response using a comparison operator.
+*/
+console.log(
+  "\u{1F327} METEOR SHOWER::::::"
+);
+
+var responses = [
+  "Call basecamp and ask for assistance, but they are far away",
+  "Do nothing, just push through",
+  "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+  "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+];
+
+// write your code here, use the console log beneath
+// console.log("\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+
+var numberOfResponses = responses.length;
+
+for (var i = 0; i <= numberOfResponses; i++) {
+  if (responses[i] === numberOfResponses[2]) {
+    console.log(responses[2] + " " + "\u{1F6F8} Swoosh -> -> Ship is steered to safety!");
+  }
+}
+
+/*
+ 4. ALIENS SPACE CRAFT ENCROACHING::::::
+    Alien craft approaches its an enemy for sure...you need to shoot it down!!
+    a) convert the pseudoscope to real code
+       create the required variables below and fill in the correct values and dataTypes to make
+       the if statements pass so that photonLaser fires pew pew pew.
+    b)
+*/
+console.log(
+  "\u{1F47E} ALIENS SPACE CRAFT ENCROACHING::::::"
+);
+
+// a.
+var enemyCraft = true;
+var photonLaser = true;
+var photonLaserEnabled = true;
+var bullets = 100;
+
+
+if (enemyCraft && photonLaser && photonLaserEnabled === true) {
```

Comment:
great.

this 
```js
if (enemyCraft && photonLaser && photonLaserEnabled ) {
```
is the same as:
```js
if (enemyCraft && photonLaser && photonLaserEnabled === true) {
```

because `photonLaserEnabled` returns a boolean, it will resolve to true or false.